rubocop-klaxit/cops/no_undefined_variable_usage.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    class NoUndefinedVariableUsage < Cop
```

Comment:
We're definitetly missing a large comment there indicating what we are looking for!